runhelm.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+export GRAFANA_DATASOURCE_PASSWORD=$(oc get secret grafana-datasources -n openshift-monitoring -o jsonpath='{.data.prometheus\.yaml}' | base64 -d | jq .datasources[0].basicAuthPassword | sed 's/"//g' )
+export PROMETHEUS_HTPASSWD_AUTH=$(oc get secret prometheus-k8s-htpasswd -n openshift-monitoring -o jsonpath='{.data.auth}')
+
+#Allow passing a values file to override helm variables.
+if [ "$1" == "--values" ] && [ "x" != "x$2" ]; then
+    EXTRA_VALUES="$1 $2"
+fi
+
+helm template \
```

Comment:
please ensure that the script exits with a non-zero value when any steps fail.

Reply:
updated.